.github/workflows/new-cfe-notifications.yml

```diff
@@ -31,33 +31,46 @@ jobs:
     if: github.event.label.name == 'CFE Approved'
     runs-on: ubuntu-20.04
     steps:
-      - name: Extract PR URL from issue body
+      - name: Extract PR info from issue body
         id: extract-pr
         run: |
-          PR_URL=$(echo "${{ github.event.issue.body }}" | grep -oP 'https://github.com/[^/]+/[^/]+/pull/\d+')
+          BODY="${{ github.event.issue.body }}"
 
-          if [[ -z "$PR_URL" ]]; then
-            echo "No valid PR URL found in the issue body. Exiting."
-            exit 1
+          PR_NUMBER=$(echo "$BODY" | grep -oP 'pull\/\K\d+')
```

Comment:
echoing $BODY is not great as mentioned in above comment.

Reply:
https://github.com/woocommerce/woocommerce/pull/52002/commits/f88df7ac30a02cf2f5fe23d86f34096969f2371e